tests/golden/copy-command/copy-command.bats

```diff
@@ -277,3 +277,41 @@ EOF
     b - [2000-01-01 01:01:01]
 EOF
 }
+
+@test "copy an entry to itself" {
+  coffer create /a
+
+  run coffer copy -f /a /a
+
+  assert_failure
+  assert_output "[ERROR] '/a' and '/a' are the same path."
+
+  run cleanOutput
```

Comment:
We don't need `cleanOutput` here. It just replaces all timestamps in the output

Reply:
Ok, removed